cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetectorManagerTest.java

```diff
@@ -95,12 +95,14 @@ private static void startRunnableDetectors(ScheduledExecutorService mockDetector
                                              GoalViolationDetector mockGoalViolationDetector,
                                              MetricAnomalyDetector mockMetricAnomalyDetector,
                                              DiskFailureDetector mockDiskFailureDetector,
+                                             BrokerFailureDetector mockBrokerFailureDetector,
```

Comment:
Should we update `BrokerFailureIntegrationTest` or add another integ. test for the adminClient-based broker failure detector?